packages/lit-dev-content/rollup.config.js

```diff
@@ -71,7 +71,8 @@ export default [
           relative.startsWith('lit/') ||
           relative.startsWith('lit-html/') ||
           relative.startsWith('lit-element/') ||
-          relative.startsWith('@lit/reactive-element/')
+          relative.startsWith('@lit/reactive-element/') ||
+          relative.startsWith('@lit-labs/ssr-client/')
```

Comment:
TBH don't feel that lit-labs/ssr-client is a big deal to see in the network tab. can you remove please or does it break something?

Reply:
Great question! Without this change it breaks production. Augustine figured this one out. Because we no longer ship hydration helpers in `lit`, or `lit-element`, it needs to be pulled into this chunk from the ssr-client.

Removing this results in https://pr1114-9f51c97---lit-dev-5ftespv5na-uc.a.run.app/ (look at search button)